pandas/tseries/offsets.py

```diff
@@ -572,9 +591,26 @@ def __init__(self, start='09:00', end='17:00', offset=timedelta(0)):
         kwds = {'offset': offset}
         self.start = kwds['start'] = _validate_business_time(start)
         self.end = kwds['end'] = _validate_business_time(end)
-        self.kwds = kwds
+        self.kwds.update(kwds)
         self._offset = offset
 
+    @cache_readonly
+    def next_bday(self):
+        # used for moving to next businessday
```

Comment:
If you're going to discuss what the function does, let's use a doc-string instead of a comment.

Reply:
This comment is retained from the existing version of the method, not sure it merits a full-on docstring.  But will do.